BGL/doc/BGL/PackageDescription.txt

```diff
@@ -708,6 +708,7 @@ user might encounter.
 ## I/O Functions ##
 - \link PkgBGLIOFct CGAL::read_off() \endlink
 - \link PkgBGLIOFct CGAL::write_off() \endlink
+- \link PkgBGLIOFct `CGAL::write_TM_to_VTP()` \endlink
```

Comment:
Why are there quotes for `CGAL::write_TM_to_VTP()` here, but not for the other functions?